main.go

```diff
@@ -119,10 +122,13 @@ func (img *CustomImage) RetagUsingSHA() error {
 	if len(img.DockerfileExtras) == 0 {
 		source := fmt.Sprintf("%s%s@sha256:%s", dockerTransport, img.Image, img.SHA)
 		wg := sync.WaitGroup{}
-		wg.Add(2)
+		wg.Add(3)
 		// Quay
 		destination := fmt.Sprintf("%s%s/%s:%s", dockerTransport, quayURL, destinationName, destinationTag)
 		go copyImage(&wg, errorCounter, source, destination)
+		// AzureCR
```

Comment:
Shouldn't we use `oci://` transport now?

Reply:
I don't think so, no.